tests/input_flows.py

```diff
@@ -1375,13 +1376,59 @@ def input_flow_t3t1(self) -> BRGeneratorType:
         yield from click_through(self.debug, screens=8, code=B.ResetDevice)
 
         # Mnemonic phrases
-        self.mnemonics = yield from load_5_shares(self.debug)
+        self.mnemonics = yield from load_N_shares(self.debug, 5)
 
         br = yield  # safety warning
         assert br.code == B.Success
         self.debug.press_yes()
 
 
+class InputFlowSlip39SingleBackup(InputFlowBase):
```

Comment:
is this identical to `BasicBackup` except for the 1 ? then perhaps make the `BasicBackup` parametrized instead?

(this will then let you parametrize `test_backup_single` with different configurations sent from Suite)

Reply:
No, `BasicBackup` needs to get the number of shares and threshold as well.

Reply:
makes sense, but this one should be parametrized then